pages/terra/policy.mdx

```diff
@@ -1,319 +0,0 @@
-
-    | Application Type | Justification |
-    |------------------|---------------|
-    | Terminal Emulators | These often only work properly when run directly on the host system. |
-    | CLI/TUI applications | The UX of Flatpak isn't optimized for these types of applications. (ex. requiring the user to call Flatpak, and then the RDNN of the app) |
-    | Modded/Multi-instance game launchers (i.e Minecraft launchers) | May require access to system libraries, and can be better managed by the user when not sandboxed. For example Minecraft launchers can easily access multiple JRE installations when not sandboxed. |
-    | Other appliactions that require manual modification of data files | Requires user to modify its data files regularly, applies in the same above case. |
-    | Applications that suffer major performance or technical issues due to sandboxing (IDEs, text editors, file managers, DAWs and other applications that rely on native plugin support like VSTs or LV2, etc.) | These applications usually require systems libraries to be present to work properly. |
-    | System components | These packages simply cannot be sandboxed, as they are part of the operating system itself. |
-    | Libraries required to build or use the aforementioned packages | These libraries are required to build or use the aforementioned packages, and are usually not sandboxed. |
-    
-    Other applications that are not specified here may be allowed on a case-by-case basis.
-- You MUST have the rights or permission to redistribute the packaged software. Besides that, there is no prohibition on proprietary licenses.
-- Submissions must be maintained. We will reject submissions of unmaintained software.
-- Submissions MUST NOT be malicious.
-- Submissions that may introduce breaking changes must be included in "Terra Extras" instead via the `subrepo = "extras"` label in `anda.hcl`.
-  - Example: `terra-mesa` should be in extra as the dependency solver will resolve this package for matching "provides" (such as `mesa-libEGL(x86-64)`).
-  - Example: `libadwaita-nightly` should be in extra since it provides `pkgconfig(libadwaita-1)`, which is required by existing packages as a build dependency.
-  - Example: `yt-dlp-nightly` **can be included normally** because it has no common provides with Fedora's `yt-dlp`.
-  - If you are in doubt whether your submission should be in extra instead, ask and we will decide.
-- Submissions SHOULD include an auto-update script (`update.rhai`).
-- Submissions SHOULD be placed in the corresponding directory in `anda/`.
-  - Each folder MUST either contain folders and no source files (other files are exempted), or contain multiple files and no folders.
-  - The following directories have a higher priority. Categorize your new package in the following order:
-    1. `fonts/`
```

Comment:
the dir stuff is also gone in this pr, this needs to go somewhere for user-facing reasons